src/parser/jobs/ast/modules/CelestialIntersection.tsx

```diff
@@ -56,12 +61,12 @@ export default class CelestialIntersection extends Module {
 			this.suggestions.add(new TieredSuggestion({
 				icon: this.data.actions.CELESTIAL_INTERSECTION.icon,
 				content: <Trans id="ast.celestial-intersection.suggestion.content">
-					Use <ActionLink {...this.data.actions.CELESTIAL_INTERSECTION} /> more frequently. Frequent uses can heal or mitigate a large amount of damage over the course of a fight, potentially resulting in fewer required healing GCDs.
+					Use <DataLink action="CELESTIAL_INTERSECTION" /> more frequently. Frequent uses can heal or mitigate a large amount of damage over the course of a fight, potentially resulting in fewer required healing GCDs.
 				</Trans>,
 				tiers: WASTED_USE_TIERS,
 				value: this.uses === 0 ? 100 : usesMissed,
 				why: <Trans id="ast.celestial-intersection.suggestion.why">
-					About {usesMissed} uses of <ActionLink {...this.data.actions.CELESTIAL_INTERSECTION} /> were missed by holding it for at least a total of {this.parser.formatDuration(holdDuration)}.
+					About {usesMissed} uses of <DataLink action="CELESTIAL_INTERSECTION" /> were missed by holding it for at least a total of {this.parser.formatDuration(holdDuration)}.
 				</Trans>,
 			}))
```

Comment:
Not going to block on this comment, since you're porting a pre-existing condition, but this module could use some additional review for EW to update how it calculates missed / max uses.  It looks like the logic here is basically going for "use it whenever it's available", but the logic doesn't account for fight downtime (e.g. you can't use the skill during the giant lazor scene in e1).  If the goal really should be to use this skill on cooldown, it might be a candidate to add to a module that extends core CooldownDowntime, since that core module does account for fight downtime.  I know that certain other heal skills (including Divine Benison) intentionally don't want to use CooldownDowntime because there are cases where you don't want to spam them (aligning with fight damage, weaving considerations, etc.), so if that applies here as well then maybe we just look at getting this module to account for fight downtime in a later PR / for EW.

Reply:
yeah I saw that so definitely something to add to TODO haha
I left a comment near the calculation in question as it definitely needs to be fixed in some sort of manner
thank you for the comment!